src/player.cpp

```diff
@@ -249,5 +232,95 @@ void Player::SeekRaw(std::uint64_t pos)
 	assert(this->file != nullptr);
 
 	this->file->Seek(pos);
-	this->file->Emit(Response::Code::TIME, this->sink);
+	this->Read("/player/time/elapsed", 0);
+}
+
+// Any resource with the single child "" (empty string) is an entry.
+// These need to be looked up via Audio, not handled by Player.
+const std::multimap<std::string, std::string> Player::RESOURCES = {
+	{"/", "/control"},
+	{"/", "/player"},
+	{"/control", "/control/state"},
+	{"/control/state", ""},
+	{"/player", "/player/file"},
+	{"/player", "/player/time"},
+	{"/player/file", ""},
+	{"/player/time", "/player/time/elapsed"},
+	{"/player/time/elapsed", ""}
+};
```

Comment:
maybe, but I just wanted something up quick and dirty with little in the way of code :P